keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -188,16 +193,16 @@ This also falls inline with how in-place update of Pod is handled(https://github
 
 * **Does enabling the feature change any default behavior?**
   Allow users to reduce size of pvc in `pvc.spec.resources`. In general this was not permitted before
-  so it should not break any of existing automation. This means that if `pvc.Spec.AllocatedResources` is available it will be
+  so it should not break any of existing automation. This means that if `pvc.Status.AllocatedResources` is available it will be
   used for calculating quota.
 
 * **Can the feature be disabled once it has been enabled (i.e. can we rollback
   the enablement)?**
   Yes the feature gate can be disabled once enabled. The quota code will use new field to calculate quota if available. This will allow quota code to use
-  `pvc.Spec.AllocatedResources` even if feature gate is disabled.
+  `pvc.Status.AllocatedResources` even if feature gate is disabled.
```

Comment:
So this means that even if I disable this feature, it leave my cluster in a dirty state afterwards?

Reply:
@gnufied  please place the details of your GH response in the doc.  The answer needs to be represented here and it means the answer to this question is "No".  This feature, once enabled, cannot be disabled because the cluster will not return to the same state as before it was enabled.